tests/testSpatialCell.py

```diff
@@ -333,22 +333,12 @@ def testInsertCandidate(self):
 #-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
```

Comment:
Please change `self.assertRaises(pexExcept.NotFoundError, tst)` at line 120 to `with self.assertRaises(...)` and delete the definition of `tst` (just move the body of that function to the body of the with statement)